src/NzbDrone.Core.Test/Download/DownloadClientTests/QBittorrentTests/QBittorrentFixture.cs

```diff
@@ -155,6 +155,20 @@ private void GivenTorrentFiles(string hash, List<QBittorrentTorrentFile> files)
                 .Returns(files);
         }
 
+        private void GivenStalledPolicy(IEnumerable<QBittorrentTorrent> torrents)
+        {
+            foreach (var torrent in torrents)
+            {
+                Mocker.GetMock<IQBittorrentStalledPolicy>()
+                    .Setup(s => s.Apply(torrent, It.IsAny<DownloadClientItem>(), It.IsAny<QBittorrentSettings>()))
+                    .Callback<QBittorrentTorrent, DownloadClientItem, QBittorrentSettings>((torr, ditem, _) =>
+                    {
+                        ditem.Status = DownloadItemStatus.Warning;
+                        ditem.Message = $"{torr.Name} Processed by stale download policy";
```

Comment:
Would rather explicitly check for the mock being called, but it seems to go against the grain of the other tests in this unit.